contracts/ERC1238/extensions/ERC1238Holdable.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../ERC1238.sol";
+import "./IERC1238Holdable.sol";
+import "./IERC1238Holder.sol";
+
+/**
+ * @dev Proposal for ERC1238 tokens extension that allow addresses
+ * to hold tokens on behalf of others.
+ */
+abstract contract ERC1238Holdable is IERC1238Holdable, ERC1238 {
+    using Address for address;
+
+    // Mapping holder => id => balance
+    mapping(address => mapping(uint256 => uint256)) private _heldBalances;
```

Comment:
Should we add another mapping for discoverability?

As a user/application I would want an easy way to check who might be a holder of a particular token. This mapping just gives us the balances of a specific holder we might be interested in.